aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -1388,12 +1386,12 @@ impl AptosVM {
 
     /// Deserialize a module bundle.
     fn deserialize_module_bundle(&self, modules: &ModuleBundle) -> VMResult<Vec<CompiledModule>> {
-        let max_version = get_max_binary_format_version(self.features(), None);
```

Comment:
@vgao1996 this is a bug, right? We default to 5 here because of passing None, but instead should select 5,6,7 based on feature flags